Server/src/routes/poi.route.ts

```diff
@@ -196,18 +197,23 @@ export class PoiRoute {
 		}
 
 		const track = (await database.tracks.getById(userData.trackId)) ?? undefined
```

Comment:
I think this null-ish coercion (`??`) is now a bit superfluous. If we return if this fails, we do not need to first convert `null` to `undefined`